api/src/test/resources/application.properties

```diff
@@ -4,7 +4,7 @@ spring.datasource.username=sa
 spring.datasource.password=sa
 spring.jpa.hibernate.ddl-auto=create-drop
 spring.jpa.properties.hibernate.dialect=org.pmiops.workbench.cdr.CommonTestDialect
-
+liquibase.enabled=false
```

Comment:
Is this for speed or correctness? Are we still running the liquibase migrations regularly?

Reply:
We have never loaded the liquibase migrations in the dao test. We could probably write a different test that validates the liquibase migrations. Currently the way to validate migrations if by running `./project.rb dev-up` or a sub command of this.

Reply:
OK, thanks. There might be a slight risk of an entity change or misunderstood design masking a schema issue, but I think we're OK.